LongWarOfTheChosen/Localization/XComGame.rus

```diff
@@ -4666,3 +4666,281 @@ TacticalText="<Bullet/> Плазмобластер выпускает разру
+TacticalText="<Bullet/> Оружие Гаусса автоматически проникает (игнорирует) 1 броню.<br/><Bullet/> Дезинтегратор частиц is the Coil equivalent of the Autocannon."
+AbilityDescName="Дезинтегратор частиц"
+
+; End Translated
+
+
+
+
+; LWOTC Translated
+BriefSummary="Солдает получает <Ability:TRIGGER_BSC_AIM_BONUS/> бонус к точности ответных выстрелов из основого оружия."
+; End Translated
+
+; LWOTC Translated
+BriefSummary="Солдает получает <Ability:TRIGGER_ADV_AIM_BONUS/> бонус к точности ответных выстрелов из основого оружия."
+; End Translated
+
+; LWOTC Translated
+BriefSummary="Солдает получает <Ability:TRIGGER_SUP_AIM_BONUS/> бонус к точности ответных выстрелов из основого оружия."
+; End Translated
+
+
+;LWOTC needs translation
+
+[MissDamageUpgrade_Bsc X2WeaponUpgradeTemplate]
+FriendlyName="Рукоятка"
+FriendlyNamePlural="Рукоятки"
+BriefSummary="Промахи из основого оружия с 20% шансом могут стать царапиной."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Adv X2WeaponUpgradeTemplate]
+FriendlyName="Улучшенная рукоятка"
+FriendlyNamePlural="Улучшенные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 25% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Sup X2WeaponUpgradeTemplate]
+FriendlyName="Элитная рукоятка"
+FriendlyNamePlural="Элитные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 30% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+; End Translation
+
+;LWOTC translated
+
+[MissDamageUpgrade_Bsc X2WeaponUpgradeTemplate]
+FriendlyName="рукоятка"
+FriendlyNamePlural="Рукоятки"
+BriefSummary="Промахи из основого оружия с 20% шансом могут стать царапиной."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Adv X2WeaponUpgradeTemplate]
+FriendlyName="Улучшенная рукоятка"
+FriendlyNamePlural="Улучшенные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 25% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+[MissDamageUpgrade_Sup X2WeaponUpgradeTemplate]
+FriendlyName="Элитная рукоятка"
+FriendlyNamePlural="Элитные рукоятки"
+BriefSummary="Промахи из основного оружия могут стать царапиной с 30% шансом."
+TinySummary="Промахи могут царапать."
+LootTooltip="Эта установленная модификация оружия делает промахи царапинами."
+
+; End Translated
+
+;LWOTC needs translation
+
+[CritUpgrade_Bsc X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%."
+
+[CritUpgrade_Adv X2WeaponUpgradeTemplate]
+BriefSummary="Increase critical hit chance by +<Ability:CRITINCREASE/>%."
+
+; End Translation
```

Comment:
These looks like English to me :smile:

Reply:
well idk what was it but now cannot find it

Reply:
found, i was translating another branch xd

Idk, it was marked to translate. Ok will try find in .int